frontend/src/components/activity/DaySwitcher.vue

```diff
@@ -115,15 +115,21 @@ export default {
 }
 </script>
 
+<!-- these styles should apply to inner elements -->
+<!-- eslint-disable-next-line vue-scoped-css/enforce-style-type -->
 <style>
 .basis-num {
   flex-basis: 2.5ch;
 }
 
+/* .e-day-switcher__menu is in the <e-select> tag */
```

Comment:
shouldn't it then be a scoped deep selector?

Reply:
you tell me.
I have not much experience in structuring (s)css or vue.
I just try to put scoped where the linter says i should (which is everywhere).
If this breaks something, i suppress the lint warning, and hope that someone with more experience really improves it in the future.